package io.github.genie.sql.core.exception;

public class BeanReflectiveException extends RuntimeException {

    public BeanReflectiveException(Throwable cause) {
        super(cause);
    }

}
